src/components/screens/monitor/delegates/delegates.js

```diff
@@ -101,19 +114,35 @@ const Delegates = ({
         className: 'standby',
       },
     ],
-    active: filters.tab,
-    onClick: ({ value }) => applyFilters({ ...filters, tab: value }),
+    active: activeTab,
+    onClick: ({ value }) => setActiveTab(value),
   };
 
-  const canLoadMore = filters.tab === 'active'
+  const canLoadMore = activeTab === 'active'
     ? false
-    : !!delegates.data.length && delegates.data.length % DEFAULT_LIMIT === 0;
+    : !!standByDelegates.data.length && standByDelegates.data.length % DEFAULT_LIMIT === 0;
 
   const getRowLink = delegate => `${routes.accounts.pathPrefix}${routes.accounts.path}/${delegate.address}`;
 
+  const chartDelegatesForging = delegates;
+
+  delegates = activeTab === 'active'
+    ? {
+      ...delegates,
+      data: delegates.data.filter(d => d.username.includes(filters.search || '')),
```

Comment:
What do you expect from this filter?
as long as `d.username` is a string, `d.username.includes(filters.search || '')` retruns true no matter what. and if you meant to filter undefined values, this is not the best logic for it.

Reply:
we can keep this based on what we discuss